### ISSUE TITLE
Enable Lucene Searching over all Repositories

### ISSUE THREAD

**Issue by gitblit** [gitblit/gitblit#829]
Originally reported on Google Code with ID 533

```
Hi Guys,

i'd like to be able to have a Link to Gitblit where i can initiate a Lucene Search
over every indexed Repository for a Commit Comment.

Basically i'd like to add Links to my Issue Tracking System to query the Source Repo
for Commits related to the Issue ID - kinda like reverse .gitbugtraq-Integration.

Right now i can use the existing Lucene Search with a few Post-Parameters (repositories
& query) but since my Issue Tracking has no clue which Repo the sources are in, i'd
have only the query.

So in short: on the search-Site i'd like the behaviour when not choosing any repos
to search in to search in all Repos instead of displaying a Error Message saying i
need to choose a Repo.
```

Reported by `friesenschrauber` on 2014-11-12 18:31:27
